doc/user/spark.md

```diff
@@ -208,7 +228,7 @@ df = spark.read.json("daos://default:1/people.json")
 ### Configure Hadoop to Use DAOS
 
 Edit `$HADOOP_HOME/etc/hadoop/core-site.xml` to change fs.defaultFS to
-`daos://default:1` or "daos://uns/\<your path\>". Then append below configuration
+`daos:///` or "daos://uns/\<your path\>". Then append below configuration
```

Comment:
Please update the URI in "Access DAOS in Spark" section. `df = spark.read.json("daos://default:1/people.json")`

Reply:
updated

Reply:
Can we say something like this?
Edit `$HADOOP_HOME/etc/hadoop/core-site.xml` to change fs.defaultFS to
`daos:///`. It is not recommended to set `fs.defaultFS` to a DAOS UNS path because .... (move your explanation here).

Reply:
changed